### PR TITLE
PFM-2579 :: Create Organisation Static screen

### PR DESCRIPTION
**Pull request by prachirane-eGov**

### CODE REVIEW

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/Masters/src/pages/employee/index.js [hunk 1]
```diff
@@ -75,6 +76,7 @@ const App = ({ path }) => {
           <MastersBreadCrumb location={location} />
         </React.Fragment>
         <PrivateRoute path={`${path}/search-masters`} component={() => <SearchMasters parentRoute={path}/>} />
+        <PrivateRoute path={`${path}/create-masters`} component={() => <CreateMasters parentRoute={path}/>} />
```
Comment (jagankumar-egov) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/Masters/src/pages/employee/index.js:79]: @prachirane-eGov  there is a route already for create org

Comment (jagankumar-egov) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/Masters/src/pages/employee/index.js:79]: use the same , also this is not master

Comment (jagankumar-egov) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/Masters/src/pages/employee/index.js:79]: mdms configured with the same

Comment (prachirane-eGov) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/Masters/src/pages/employee/index.js:79]: Not updated the route, just renamed older component and added new component on same route

Comment (jagankumar-egov) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/Masters/src/pages/employee/index.js:79]: ok

**Review by jagankumar-egov — APPROVED** [review]

**Pull request merged**